tests/queries/tests.py

```diff
@@ -1172,6 +1173,17 @@ def test_lookup_constraint_fielderror(self):
         with self.assertRaisesMessage(FieldError, msg):
             Tag.objects.filter(unknown_field__name='generic')
 
+    def test_common_mixed_case_foreign_keys(self):
```

Comment:
I'd add a docstring along the lines of "verify that a valid query is generated when fields fetched from joined tables include FKs whose name only differ by case"

Reply:
The coding guidelines suggest to omit prefixes like "Verify that" since all tests verify things, otherwise the patch looks fine.

Reply:
Thanks. I updated the docstring,